packages/lisk-core/src/cli/actions/start.js

```diff
@@ -0,0 +1,11 @@
+const config = require('../../helpers/config');
+const Controller = require('../../controller');
+const ComponentFactory = require('../../factories/components');
+
+const logger = ComponentFactory.create('logger', config.components.logger);
+
+module.exports = async function start(options = {}) {
+	logger.info('CMD start');
+	logger.debug('CMD start called with options', options);
+	return Controller.start(options);
```

Comment:
Having a static method that creates it's own instance is adding more complexity.

```js
const controller = new Controller();
try {
 // We can do all one time initialisations here. Also config validation. 
  await bus.setup();
  await controller.start(options);
} catch (err) {
  // This will resolve unhandled promise rejection warning!
  console.error(err);
}
```